pegasus/sites.v3/hourofcode.com/public/how-to/index-draft.haml

```diff
@@ -8,7 +8,7 @@ theme: how_to_guide
 = view :"how-to/how_to_banner", nav_audience: "guide_nav_educators"
 
 -# Intro section
-= view :"how-to/how_to_intro", intro_header: hoc_s(:hoc_how_to_educators_intro_heading), intro_desc: hoc_s(:hoc_how_to_educators_intro_desc)
+= view :"how-to/how_to_intro", intro_header: hoc_s(:hoc_how_to_educators_intro_heading), intro_desc: hoc_s(:hoc_how_to_educators_intro_desc), intro_desc_02: ""
```

Comment:
All of the other how-to guides have two paragraphs in this section so I added this empty string on the Educators (/index) one here so I could include an `intro_desc_02` variable on the `how_to_intro` view.